tests/unit/amo/components/Errors/TestNotFound.js

```diff
@@ -32,10 +33,8 @@ describe(__filename, () => {
   it('renders a not found error', () => {
     const root = render();
 
-    expect(root.find('.Errors')).toHaveProp(
-      'header',
-      'Oops! We can’t find that page',
-    );
+    expect(root.find(ErrorComponent)).toHaveProp('code', 404);
+    expect(root).toHaveProp('header', 'Oops! We can’t find that page');
```

Comment:
This should be `expect(root.find(ErrorComponent)).toHaveProp('header', 'Oops! We can’t find that page');`